base/applications/mspaint/winproc.cpp

```diff
@@ -1069,6 +1069,13 @@ LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
             fullscreenWindow.DoCreate();
             fullscreenWindow.ShowWindow(SW_SHOWMAXIMIZED);
             break;
+
+        case IDM_CTRL_PLUS:
+            toolsModel.ControlPlusMinus(FALSE);
+            break;
+        case IDM_CTRL_MINUS:
+            toolsModel.ControlPlusMinus(TRUE);
```

Comment:
Those need to name change as well.

Reply:
What name?
`SpecialAction`?

Reply:
`AdjustPlusMinus`

Reply:
Hmmmm... now that you're asking: Do you think the "CtrlPlusMinus" thing can do different things depending on which tool is currently selected?

Reply:
`SpecialTweak`

Reply:
Yes, it can do different jobs.
https://github.com/katahiromz/RPaint/issues/1

Reply:
Renamed as `SpecialTweak`.

Reply:
OK!